test/linalg/dense.jl

```diff
@@ -454,31 +454,55 @@ end
         A5  = convert(Matrix{elty}, [1 1 0 1; 0 1 1 0; 0 0 1 1; 1 0 0 1])
         @test expm(logm(A5)) ≈ A5
 
-        A6  = convert(Matrix{elty}, [-5 2 0 0 ; 1/2 -7 3 0; 0 1/3 -9 4; 0 0 1/4 -11])
-        @test expm(logm(A6)) ≈ A6
```

Comment:
so the new algorithm returns a different root? should we test the new result instead?